sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -54,34 +55,32 @@
       "ky",
       "E"
     ],
-    "slow_axes": "@attrs:metadata/instrument/analyzer/slow_axes",
+    "slow_axes": "t.b.d.",
     "energy_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/energy_resolution",
+      "resolution": 100,
       "resolution/@units": "meV",
       "physical_quantity": "energy",
       "type": "estimated"
     },
     "momentum_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/momentum_resolution",
+      "resolution": 0.08,
       "resolution/@units": "1/angstrom",
       "physical_quantity": "momentum",
       "type": "estimated"
     },
     "spatial_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/spatial_resolution",
+      "resolution": 1,
       "resolution/@units": "µm",
       "physical_quantity": "length",
       "type": "estimated"
     }
   },
   "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/COLLECTIONCOLUMN[collectioncolumn]": {
     "projection": "@attrs:metadata/scientificMetadata/Collection/projection",
-    "scheme": "None",
+    "scheme": "momentum dispersive",
```

Comment:
Should/could also be entered automatically depending on lens mode.

Reply:
This also added automatically into ScIcat during last beamtime

Reply:
Then get it from sciCat

Reply:
In the description for NXcollectioncolumn/scheme, it has these options:
standard, deflector, PEEM, momentum microscope, etc.
For us only PEEM, momentum microscope valid or?
Can I am anyway use momentum dispersive as it was already ingested during last beamtimes or do I need to convert in one of those above?

Reply:
This is outdated, in the new definitions it is only angular dispersive, momentum dispersive, non-dispersive or spatial dispersive:
https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXmpes.html#nxmpes
This should still be reflected in the base class, as I just noticed.

Reply:
Ok,
I was looking directly for NXcollectioncolumn
https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXcollectioncolumn.html#nxcollectioncolumn
and it still has old options.